### PR TITLE
JAMES-1774 Remove duplicated AI mailets

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by mbaechler** [comment]
:+1:

**Comment by mbaechler** [comment]
:+1:

**Comment by chibenwa** [comment]
test this please

**Comment by aduprat** [comment]
:+1:

**Comment by rouazana** [comment]
how did you choose which version to take?

**Comment by chibenwa** [comment]
I kept these in mailet/ai where they belongs.

**Comment by rouazana** [comment]
either I don't understand your answer, or you didn't understand my question :)
The files you are removing are different for the ones you are keeping. So how did you choose to keep one version and not the other one? And why there is no merge between the 2 versions?

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please
